src/Body.jl

```diff
@@ -73,4 +76,28 @@ measure_sdf!(a::AbstractArray,body::AbstractBody,t=0;kwargs...) = @inside a[I] =
 Use for a simulation without a body.
 """
 struct NoBody <: AbstractBody end
-function measure!(a::Flow,body::NoBody;t=0,ϵ=1) end
+measure(::NoBody,x::AbstractVector,args...;kwargs...)=(Inf,zero(x),zero(x))
+function measure!(::Flow,::NoBody;kwargs...) end # skip measure! entirely
+
+"""
+    SetBody
+
+Body defined as a lazy set operation on two `AbstractBody`s.
+The operations are only evaluated when `measure`d.
+"""
+struct SetBody{O<:Function,Ta<:AbstractBody,Tb<:AbstractBody} <: AbstractBody
```

Comment:
Should we make it mutable to allow to change/update one of the body like `DynamicNurbs`?

Reply:
Maybe so! I didn't try that yet.